### PR TITLE
Added class to obtain checksums and package name

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #440 
Added class to obtain checksum and package name, will use it to figure out what packages were added/removed from the repository.

### CODE REVIEW

File: src/main/java/com/artipie/rpm/asto/AstoChecksumAndName.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            .to(SingleInterop.get());
+    }
+
+    /**
+     * Key part without initial part.
+     * @param key Initial key
+     * @param item Item key
+     * @return Item string name name
+     */
+    private static String keyPart(final Key key, final Key item) {
```
Comment (genryxy) [inline src/main/java/com/artipie/rpm/asto/AstoChecksumAndName.java:70]: I'd suggest to rename first parameter

File: src/main/java/com/artipie/rpm/asto/AstoChecksumAndName.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+import io.reactivex.Single;
+import java.util.Map;
+import java.util.concurrent.CompletionStage;
+import org.cactoos.map.MapEntry;
+
+/**
+ * Checksums and names of the storage items.
+ * @since 1.10
+ */
+public final class AstoChecksumAndName {
```
Comment (genryxy) [inline src/main/java/com/artipie/rpm/asto/AstoChecksumAndName.java:23]: let's make this class package private

File: src/main/java/com/artipie/rpm/asto/AstoChecksumAndName.java [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+     * Key part without initial part.
+     * @param key Initial key
+     * @param item Item key
+     * @return Item string name name
+     */
+    private static String keyPart(final Key key, final Key item) {
+        String res = item.string();
+        if (!key.equals(Key.ROOT)) {
+            res = item.string().substring(key.string().length() + 1);
+        }
```
Comment (genryxy) [inline src/main/java/com/artipie/rpm/asto/AstoChecksumAndName.java:74]: probably add todo here to use class from `artipie/asto` for excluding a specific part of the key

**Review by genryxy — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, several comments

File: src/main/java/com/artipie/rpm/asto/AstoChecksumAndName.java (hunk 3 above)
Comment (olenagerasimova) [reply src/main/java/com/artipie/rpm/asto/AstoChecksumAndName.java:74]: There are already several places to use this new class without todos, so I simply do it when `asto` will be released

**Review by genryxy — APPROVED** [review]
@olenagerasimova thanks, looks good

**Pull request merged**